app/helpers/layout_helper.rb

```diff
@@ -194,11 +194,19 @@ def time_zone_select_f(f, attr, default_timezone, select_options = {}, html_opti
     end
   end
 
-  def selectable_f(f, attr, array, select_options = {}, html_options = {})
+  def selectable_f(f, attr, array, select_options = {}, html_options = {}, btn_options = {})
+    btn_group = btn_options.delete(:btn_group)
+    btn_lbl = btn_options.delete(:btn_lbl)
```

Comment:
Avoid using the abbreviation in the hash key, call it `button_label` or simply `label` for consistency with other options.